### DISCUSSION TITLE
[plex error]: Your library seems empty workaround

### DISCUSSION THREAD

**Discussion by MatthewSimunec** [itsToggle/plex_debrid#595] [General]
Hi all, 

Is there a way I can I keep my library empty and add new watchlists to RD?

I like to keep it clean/empty by deleting stuff I've watched. 
It seems I always need to keep 1 movie and TV show in the library to allow Plex to pull new stuff, otherwise it errors out. 

If I accidentally delete everything, I need to open Stremio to cache new stuff to library to get plex RD to work around, kinda annoying!

Anyone figured out a work around?

Thanks

**Comment by Sandspit87** [comment]
Can you explain your workaround with stremio? I'm in the same situation.
